app/src/main/java/org/oppia/app/viewmodel/ViewModelBridgeFactory.kt

```diff
@@ -2,16 +2,16 @@ package org.oppia.app.viewmodel
 
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
+import java.security.Provider
```

Comment:
Please fix this import--this one is unused, and we should import the javax.inject.Provider import from below.